### ISSUE TITLE
make this repo private

### ISSUE THREAD

**Issue by anujpuri72** [TanmayAmbadkar/ResaleValuePrediction#1]
This repo is public so other user users can see this. 
Also upload there project report ASAP @TanmayAmbadkar

**Comment by thepushkarp** [comment]
Yeah @anujpuri72 @TanmayAmbadkar. Somebody asked for a project idea for the assignment. This seems a good repo for them to check out :thinking:. Even the code is available to start with.

**Comment by thepushkarp** [comment]
Don't make this private. How else would others copy your assignment then?

**Comment by aashutoshrathi** [comment]
I think you also need to remove your DB config, else **anyone** can wipe it out.

**Comment by AmanRaj1608** [comment]
Keep it public but just remove this. You can use env variables.

https://github.com/TanmayAmbadkar/ResaleValuePrediction/blob/master/my.cnf

**Comment by AmanRaj1608** [comment]
I made my project repo private and now I can not use Travis ci 😢. 
Don't be like me. Keep it public.

**Comment by thepushkarp** [comment]
@AmanRaj1608 Does CI on GitHub Actions support Private Repos?

**Comment by thepushkarp** [comment]
Like Circle CI?

**Comment by AmanRaj1608** [comment]
Oh, I thought all ci was paid for private repo's. 
Thanks, bro 🙏
